site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -167,6 +167,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.controller.nodeSelector`                         | Controller [node labels][nodeSelector] for pod assignment                                                                                                                                                                                                                                                        | `{}`                                      |
 | `agones.controller.tolerations`                          | Controller [toleration][toleration] labels for pod assignment                                                                                                                                                                                                                                                    | `[]`                                      |
 | `agones.controller.affinity`                             | Controller [affinity][affinity] settings for pod assignment                                                                                                                                                                                                                                                      | `{}`                                      |
+| `agones.controller.labels`                               | [Labels][labels] added to the Agones controller pods                                                                                                                                                                                                                                                             | `{}`                                      |
```

Comment:
Since we publish docs on merge, we'll need you to use a [feature shortcode](https://agones.dev/site/docs/contribute/#within-a-page) to hide these fields until next release.

You will likely have to make a copy of the table for each version (doing this in the middle of a table tends to render badly).